### ISSUE TITLE
[Feature] Add reminder to save before quitting

### ISSUE THREAD

**Issue by Huy-Ngo** [DevinR528/forget#2]
One may forgot to save the list before leaving, which leads one to lose the added item. A reminder to save may help.

**Comment by DevinR528** [comment]
Yeah, I agree this would be a great feature to have! If you would be interested in submitting a PR that would be awesome! Otherwise, I will try to get to this when I have time.

**Comment by Huy-Ngo** [comment]
I'm new to Rust so I'm not sure I can do it but I'll try to study the code.
